### ISSUE TITLE
XWFIM - HTTP 404 error message for MPlayer x64.zip file (file not found here)

### ISSUE THREAD

**Issue by 4n6G33K** [EricZimmerman/Issues#204]
Hi Eric,

Hope all is well as always.  Just wanted to let you know that there is a minor issue w/ XWFIM that I was finally able to diagnose today (haven't been able to download the latest version of X-Ways because it keeps crapping out w/ a 404 error).

I was able to determine that in the very last step of your process, you are making a call to the MPlayer folder on the X-Ways portal to a version that no longer exists in that location - it is indeed in the history folder, but the lastest version is 2018 and you are attempting to download the 2015 file in the current version of XWFIM, hence it keeps throwing this error.

Just thought you might want to know, I was able to confirm the file is not in that location on the X-Ways site and found only the 2108 MPlayer zip file now sitting at the root instead.  

Cheers,
M.

**Comment by EricZimmerman** [comment]
Are you sure you have the latest version from my website? The software doesn't auto update anymore and as far as I know I just did an update last week for x-rays and everything works fine

**Comment by 4n6G33K** [comment]
Yep, positive I had the latest version.  I even had a colleague test it on his forensic workstation and he ended up with the same result.

**Comment by EricZimmerman** [comment]
as far as i can tell you are mistaken, somehow. did you check to use the 32 bit mplayer? if so, thats the issue. uncheck it. i will be removing that option in the future

XWFIM 2.1 downloaded today produces the same files as manually downloading mplayer from xwf website as shown below.

No errors with default options

![image](https://user-images.githubusercontent.com/4265254/234354303-70e043c0-b546-4ac5-9b1b-b9a1b5aaaad5.png)


![image](https://user-images.githubusercontent.com/4265254/234354066-1e239eca-d16a-41b1-9b4a-037355a5589d.png)

**Comment by EricZimmerman** [comment]
i just fixed the path to the 32 bit version and will be uploading a new build here in 10 mins

**Comment by 4n6G33K** [comment]
Thanks Eric, and to your earlier question about the 32-bit MPlayer, yes we absolutely made certain that we were attempting to download the correct one.  That’s how we identified the problem to begin with when we found it was looking in a directory that was no longer present (nor was that file resident).

 

Appreciate the feedback and the update.  

 

Hope all is well as always my friend, stay safe !

 

Cheers,

M.
